src/models/scenario.ts

```
@@ -0,0 +1,47 @@
+    } catch (error: unknown) {
+      console.error(`Error fetching scenario ${_id}:`, error);
+      throw error;
+    }
+  }
+
+  static async findAll(
+    this: ReturnModelType<typeof Scenario>
+  ): Promise<Scenario[] | null> {
+    try {
+      await connectToDatabase();
+      return await this.find({}).lean();
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)